<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>caseinfodialog</class>
 <widget class="QDialog" name="caseinfodialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>323</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>811</width>
    <height>323</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>811</width>
    <height>323</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QScrollBar
{
    width:8px;
}
 QSlider::groove
{
    border:none;
    background: #3F444A;
    margin-left:12px;
    margin-right:12px;
}
QSlider::handle
{
    margin-left:-12px;
    margin-right:-12px;
    height:12px;
    background: #1167CF;
    border-radius: 4px;
}
#widget{
border-bottom:2px solid #282C36;
background:#1C212D;
}
#caseinfodialog
{
background:#1C212D;
}
#widget_2 QLabel{
font-size: 18px;
font-family: &quot;思源黑体 CN Medium&quot;;
font-weight: 500;
color: #FFFFFF;
background:transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBar" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>55</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>55</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">#widget_2
{
border-bottom:2px solid #282C36;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>48</number>
         </property>
         <property name="topMargin">
          <number>21</number>
         </property>
         <property name="rightMargin">
          <number>48</number>
         </property>
         <property name="bottomMargin">
          <number>21</number>
         </property>
         <property name="horizontalSpacing">
          <number>65</number>
         </property>
         <property name="verticalSpacing">
          <number>23</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>思源黑体 CN Medium</family>
               <pointsize>-1</pointsize>
               <weight>62</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>患者ID：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_patientID">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>231</width>
               <height>33</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>231</width>
               <height>33</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string>请输入患者ID</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>思源黑体 CN Medium</family>
               <pointsize>-1</pointsize>
               <weight>62</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>患者性别：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_patientGender">
             <property name="minimumSize">
              <size>
               <width>231</width>
               <height>33</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>231</width>
               <height>33</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <item>
              <property name="text">
               <string>男</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>女</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>思源黑体 CN Medium</family>
               <pointsize>-1</pointsize>
               <weight>62</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>患者姓名：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_patientName">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>231</width>
               <height>33</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>231</width>
               <height>33</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>黑体</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string>请输入患者姓名</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>思源黑体 CN Medium</family>
               <pointsize>-1</pointsize>
               <weight>62</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>患者年龄：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="lineEdit_patientAge">
             <property name="minimumSize">
              <size>
               <width>231</width>
               <height>33</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>231</width>
               <height>33</height>
              </size>
             </property>
             <property name="maximum">
              <number>150</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>思源黑体 CN Medium</family>
               <pointsize>-1</pointsize>
               <weight>62</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>医生账户：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_userid">
             <property name="minimumSize">
              <size>
               <width>231</width>
               <height>33</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>231</width>
               <height>33</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>24</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>24</number>
        </property>
        <property name="bottomMargin">
         <number>21</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_done">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>确 定</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff></normaloff></iconset>
          </property>
          <property name="class" stdset="0">
           <string>BlueButton</string>
          </property>
          <property name="TextSize" stdset="0">
           <string>18</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_close">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>34</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>关闭</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff></normaloff></iconset>
          </property>
          <property name="class" stdset="0">
           <string>BlackButton</string>
          </property>
          <property name="TextSize" stdset="0">
           <string>18</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qr.qrc"/>
 </resources>
 <connections/>
</ui>
